### PR TITLE
Trying to fix authorization problems on Windows

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Added some logging, hoping it uncovers problems

### CODE REVIEW

**Comment by cbjeukendrup** [comment]
I'm going to close this now, since Bradley and I _finally_ found what was wrong. 

TL;DR: `C:\Windows\System32\msvcr100.dll` was missing on Bradleys (virtual) Windows machine, which caused the libssl and libcrypto libraries to fail loading, which caused any https connection in MuseScore to fail, which caused downloading the access token to fail.

@DmitryArefiev So if we ever get a report of a user who is unable to log in in the app, or if the videos on the Learn page will never load, or if they always get a "unable to connect to MuseScore.com" popup when trying to publish, then their `C:\Windows\System32\msvcr100.dll` might be missing. Do you have by any chance a place where you document such "possible solutions for problems that are out of our control"?

**Comment by DmitryArefiev** [comment]
> @DmitryArefiev So if we ever get a report of a user who is unable to log in in the app, or if the videos on the Learn page will never load, or if they always get a "unable to connect to MuseScore.com" popup when trying to publish, then their `C:\Windows\System32\msvcr100.dll` might be missing. Do you have by any chance a place where you document such "possible solutions for problems that are out of our control"?

I will put this info into my internal document "User cases". Thanks!

But the best place for potential cases like that (+solution) is https://musescore.org/en/project/issues/musescore
